.github/workflows/release-manual-stable-81.yml

```diff
@@ -0,0 +1,80 @@
+name: Release Stable 8.1
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
+        description: 'The branch to perform the release on, defaults to `release-$releaseVersion`'
+        type: string
+        required: false
+        default: ''
+      releaseVersion:
+        description: 'The version to be build and released. If no releaseBranch specified, expecting `release-$releaseVersion` to already exist.'
+        type: string
+        required: true
+      nextDevelopmentVersion:
+        description: 'Next development version, e.g. 8.X.X-SNAPSHOT.'
+        type: string
+        required: true
+      isLatest:
+        description: 'Whether this is the latest release and the docker image should be tagged as camunda/zeebe:latest'
+        type: boolean
+        required: false
+        default: false
+      dryRun:
+        description: 'Whether to perform a dry release where no changes or artifacts are pushed, defaults to true.'
+        type: boolean
+        default: true
+
+concurrency:
+  # cannot use the inputs context here as on this level only the github context is accessible, see
+  # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency
+  group: ${{ github.event.inputs.releaseBranch != '' && github.event.inputs.releaseBranch || format('release-{0}', github.event.inputs.releaseVersion) }}
+  cancel-in-progress: true
+
+jobs:
+  run-release:
+    name: "Release ${{ inputs.releaseVersion }} from ${{ inputs.releaseBranch }}"
+    uses: camunda/zeebe/.github/workflows/release.yml@stable/8.1
```

Comment:
:question: couldn't that be parameterized, so we don't need multiple workflows? :thinking: 

Sorry not that experienced with gha

Reply:
I would have wished so too, unfortuantely not :( , the workflow is parsed and wired before any contexts are initialised, 
see https://github.com/orgs/community/discussions/24661

Reply:
@oleschoenburg  an alternative might be to just move `release-manual.yml` to the stable branches as well and then select the stable branches when invoking the workflow, right? 🤔 
![image](https://user-images.githubusercontent.com/209518/208616607-7a7d2f42-810e-4057-a199-06a7632f4c19.png)

Reply:
it worked out, given https://github.com/camunda/zeebe/pull/11300 & https://github.com/camunda/zeebe/pull/11301 are merged we will just have one manual release workflow 🎉